packages/block-library/src/paragraph/test/edit.native.js

```diff
@@ -12,6 +12,10 @@ import Paragraph from '../edit';
  * WordPress dependencies
  */
 jest.mock( '@wordpress/blocks' );
+jest.mock( '../../../../data/src/components/use-select', () => () => ( {
```

Comment:
After some tests trying to mock this, using `requireActual`, etc. I wasn't able to make it work due to some exports issues.

We can't use `@wordpress/data` as the path due to the symlinks we have in `Gutenberg Mobile`

There were some tests in another mock here https://github.com/WordPress/gutenberg/pull/19705#issuecomment-575262766 without any luck.

I'll keep an eye after this is merged to see how we can improve this.